### ISSUE TITLE
Webextensions don't respect cookie policy

### ISSUE THREAD

**Issue by StanGets** [mozilla/addons#895]
Hi,

Firefox 65.0 64-bit / Windows 7 64-bit

I've just noticed that Webextensions which include calling sites don't respect my general cookie policy.

IF I set [network.cookie.cookieBehavior](url) = 2 = Block all cookies :

example 1 : uBlock Origin : i.e. https://secure.fanboy.co.nz/r/fanboy-ultimate.txt
result : fanboy.co.nz has set a cookie
expected : no cookie given `network.cookie.cookieBehavior` = 2

example 2 : Feedbro : i.e. https://fileforum.betanews.com/rss
result : betanew.com has set a cookie
expected : no cookie given `network.cookie.cookieBehavior` = 2

BUT, if I explicitly make a blocking exception, in above examples,
for https://secure.fanboy.co.nz,
for https://fileforum.betanews.com
THEN there are no cookies from these sites.

In other words Webextensions consider specific site cookie permissions but not the global cookie behavior policy all in `network.cookie.cookieBehavior`

I'm aware of Webextensions required and accepted permissions upon install but I didn't believe these would prevail on Firefox's default cookie policy. I'm quite surprised. Is this a bug or is it in the Webextensions' logic? Thanks.

**Comment by wagnerand** [comment]
This repository is for the underlying add-ons.mozilla.org platform. Can you please file a bug on https://bugzilla.mozilla.org/enter_bug.cgi?product=WebExtensions&component=Untriaged ? Thanks!

**Comment by StanGets** [comment]
OK, I've filled the bug at [1525917 - Webextensions don't respect cookie policy (ME)](https://bugzilla.mozilla.org/show_bug.cgi?id=1525917)

Thanks.
